python/turbodbc/options.py

```diff
@@ -52,6 +53,10 @@ def make_options(read_buffer_size=None,
      report a size of 2 billion characters.
      Please note that this option only relates to retrieving results, not sending parameters to the
      database.
+    :param fix_unicode_allocation Some ODBC drivers report the length of the varchar/nvarchar field rather
```

Comment:
Please conform to the style of the other parameters:
1. Affects behavior/performance
2. If set to `True`, ...